src/translations/fr.json

```
@@ -1574,5 +1575,7 @@
     "qmax": "Q Max",
     "nodeAliasesRetrievingError": "Erreur lors de la récupération des alias de nœuds",
     "nodeAliasesUpdateError": "Erreur lors de la mise à jour des alias de nœuds",
-    "mustBeGreaterOrEqualToZero": "Cette valeur doit être supérieure ou égale à 0"
+    "mustBeGreaterOrEqualToZero": "Cette valeur doit être supérieure ou égale à 0",     
+    "confirmRootNetworkCreation": "L’étude {studyName}, va passer dans une visualisation adaptée à l'étude de plusieurs réseaux racines. \n\nLe changement d'interface est irréversible."
```

```suggestion
    "confirmRootNetworkCreation": "L’étude {studyName} va passer dans une visualisation adaptée à l'étude de plusieurs réseaux racines. \n\nLe changement d'interface est irréversible."
```

I know this comma is on FIGMA, but I don't think there should be one here

---

ok